tools/buildLundiumPackage.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-console */
+const path = require('path');
+
+const glob = require('glob');
+const webpack = require('webpack');
+
+const packagePath = process.cwd();
+const lundiumPackagePath = './packages/lundium';
+const relativeSrcPath = `${lundiumPackagePath}/src/components`;
+const srcPath = path.join(packagePath, relativeSrcPath);
+
+const directoryPackages = glob.sync('**/index.js', { cwd: srcPath }).map(path.dirname);
+
+const entries = directoryPackages.reduce(
+	(entries, newEntry) => ({
+		...entries,
+		[newEntry]: `${relativeSrcPath}/${newEntry}/index.js`,
+	}),
+	{}
+);
+
+const compiler = webpack({
+	mode: 'production',
+	entry: {
+		index: `${lundiumPackagePath}/src/index.js`,
+		...entries,
+	},
+	output: {
+		filename: '[name].js',
+		path: path.resolve(`${lundiumPackagePath}/dist`),
+	},
+	module: {
+		rules: [
+			{
+				test: /\.js$/,
+				exclude: /node_modules/,
+				loader: 'babel-loader',
```

Comment:
missing `options: { rootMode: "upward" }`

Reply:
I've recreated it a bit, so this will be root config, since it should be primary build of this project